TelCo.ColorCoder/ColorCoderPair.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoderClasses
+{ 
+    public class ColorGroup
```

Comment:
getMajorIndex, getMinorIndex is to generic Name....

In C# method name should be written using Pascal Convention
ex: getMajorIndex -> GetMajorIndex

Reply:
generic names are replaced with more specific names